src/Persistence/Initialization/VersionSeasonSix/ChaosMixes.cs

```diff
@@ -447,21 +439,20 @@ private ItemCrafting ThirdWingsStage2Crafting()
         var crafting = this.Context.CreateNew<ItemCrafting>();
         crafting.Name = "3rd Level Wings, Stage 2";
         crafting.Number = 39;
+        crafting.ItemCraftingHandlerClassName = typeof(ThirdWingsCrafting).FullName!;
         var craftingSettings = this.Context.CreateNew<SimpleCraftingSettings>();
-
         crafting.SimpleCraftingSettings = craftingSettings;
-        craftingSettings.Money = 5_000_000;
-        craftingSettings.MoneyPerFinalSuccessPercentage = 10000;
-        craftingSettings.SuccessPercent = 0;
+        craftingSettings.MoneyPerFinalSuccessPercentage = 200_000;
+        craftingSettings.SuccessPercent = 1;
```

Comment:
https://github.com/kyleruss/emu-server/blob/202856a74c905c203b9b2795fd161f564ca8b257/GameServer/Source/ChaosBox.cpp#L4635
According to my tests, the client always adds 1% to the calculated rate.